Integrations/integration-Netskope.yml

```diff
@@ -225,13 +225,45 @@ script:
         return {
             Type: entryTypes.note,
             ContentsFormat: formats.json,
-            Contents: retArray,
+            Contents: result.data,
             ReadableContensFormat: formats.markdown,
             HumanReadable: tableToMarkdown('Netskope Events', retArray, headers),
             EntryContext: ec
         };
     }
 
+    function now() {
+        return parseInt(new Date().getTime()/1000)
+    }
+
+    function fetchAlerts() {
+        last_run = parseInt(getLastRun())
+        if (!last_run) {
+            last_run = now() - parseInt(translateTimeperiod('Last24Hours'))
```

Comment:
add parameter to set the first fetch time, instead of hard coding it

Reply:
done